api_reference.md

```diff
@@ -120,6 +122,27 @@ thoroughly.
 
 For more details on how these were trained, see the [Toxicity and sub-attribute annotation guidelines](https://github.com/conversationai/conversationai.github.io/blob/master/crowdsourcing_annotation_schemes/toxicity_with_subattributes.md).
 
+
+#### Toxicity models for Spanish, French and German comments
+
+The following models are our first non-English TOXICITY models. Only use these if you are
+interested in testing an experimental model and are willing to change your code once the
+production models are available. These models are experimental and have not been tested as
+thoroughly as their English counterparts.
+
+*   TOXICITY_EXPERIMENTAL
+*   SEVERE_TOXICITY_EXPERIMENTAL
+*   PROFANITY_EXPERIMENTAL
+*   IDENTITY_ATTACK_EXPERIMENTAL
+*   INSULT_EXPERIMENTAL
+*   THREAT_EXPERIMENTAL
```

Comment:
Could you bold the model names with `**` to fit the same style as the other model names?